run.sh

```diff
@@ -90,7 +92,7 @@ echo "this shouldn't take more than a minute.."
 num=0
 while true; do
   num=$(wget --quiet -O - "http://$graphitemon_host:8000/render/?target=graphite-watcher.$env.num_metrics&from=-2min&until=-10s&format=raw" | sed -e 's#.*,##')
-  [ "$num" == "${total}.0" ] && break
+  (( $(echo  "$num >=  $total"|bc -l) )) && break
```

Comment:
> due to the number of metrics far exceeding the expected Total

that would be a bug in the script.
if we allow the number of metrics to be higher than what we expect, we open the opportunity for metrics we need to be missing because they can be masked by the metrics that show up but shouldn't.
the right solution is keeping the comparison to `==` or `-eq`, and using inspect-es in raintank-metric (or any tool of your preference) to find out what the extra metrics are.
to be clear, the benchmarks are designed to be run against a known "workload" created through env-load. obviously you can increase the number of metrics by manually adding endpoints and such, but you shouldn't do that to get reliable, comparable benchmarks.